### ISSUE TITLE
Issues with downloading from Reddit (Gonewild/Gonemild - Imgur)

### ISSUE THREAD

**Issue by AranciataExcess** [4pr0n/ripme#308]
**Issue**

Trying to rip galleries of users (primarily gonemild & gonewild) but it is not downloading anything. I've noticed this recently with many empty folders of newer GW/GM users with images hosted on Imgur. 

The log file inside the folders have the following errors;

**Example:**
Ripping the following user - https://www.reddit.com/user/Aleexandriaa-4/

**Empty folder:**
http://i.imgur.com/tMYldy7.png

Sample error (see attached logfile for more)

The image exists at location - http://imgur.com/FNzKnXO

```
2016-03-01 19:52:43,235 DEBUG ripper.DownloadThreadPool .initialize() Initializing Main thread pool with 5 threads
2016-03-01 19:52:44,178 DEBUG utils.RipUtils .getFilesFromURL() Checking http://imgur.com/FNzKnXO
2016-03-01 19:52:45,122 ERROR utils.RipUtils .getFilesFromURL() [!] Unable to rip URL: http://imgur.com/FNzKnXO
```

**Logfile:**
See attached file
[log.txt](https://github.com/4pr0n/ripme/files/152450/log.txt)

**Comment by 4pr0n** [comment]
Pretty sure this got fixed in another commit (should work in 1.2.10).

Regardless, I noticed that user was using `i.reddituploads.com` and the ripper was missing them, so I added support for `i.reddituploads.com` as well.

**Comment by AranciataExcess** [comment]
Confirmed working. Thanks for the fix.
